lib/create.js

```diff
@@ -0,0 +1,38 @@
+import {createHasher, hashCanonizedProof, stringToUtf8Bytes}
+  from '@digitalbazaar/di-sd-primitives';
+import {canonize} from './canonize.js';
+import {concat} from './helper.js';
+import {createVerifier} from './createVerifier.js';
+import {name} from './name.js';
+import {requiredAlgorithm} from './requiredAlgorithm.js';
+
+export function createCryptosuite({extraInformation = ''} = {}) {
+  const options = {extraInformation};
+  return {
+    name,
+    canonize,
+    requiredAlgorithm,
+    createVerifier,
+    createVerifyData: _createSignData,
+    options
+  };
+}
+
+async function _createSignData({cryptosuite, document, proof, documentLoader}) {
+  const options = {documentLoader};
+  const externalHash = cryptosuite.options.extraInformation;
```

Comment:
As mentioned in another comment, I think we should do the hashing internally to the cryptosuite.

Reply:
Fixed